src/webui/api/appcontroller.cpp

```diff
@@ -776,6 +778,9 @@ void AppController::setPreferencesAction()
 
     // Advanced settings
     // qBittorrent preferences
+    // Resume data storage type
+    if (hasKey(u"resume_data_storage_type"_qs))
+        session->setResumeDataStorageType(static_cast<BitTorrent::ResumeDataStorageType>(it.value().toInt()));
```

Comment:
Would it be hard to implement to pass it in using string representation?

Reply:
Done.